File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/PlaybackTracking.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class PlaybackTracking{

	@SerializedName("videostatsWatchtimeUrl")
	private VideostatsWatchtimeUrl videostatsWatchtimeUrl;

	@SerializedName("videostatsDelayplayUrl")
	private VideostatsDelayplayUrl videostatsDelayplayUrl;

	@SerializedName("qoeUrl")
	private QoeUrl qoeUrl;

	@SerializedName("youtubeRemarketingUrl")
	private YoutubeRemarketingUrl youtubeRemarketingUrl;

	@SerializedName("googleRemarketingUrl")
	private GoogleRemarketingUrl googleRemarketingUrl;

	@SerializedName("setAwesomeUrl")
	private SetAwesomeUrl setAwesomeUrl;

	@SerializedName("videostatsPlaybackUrl")
	private VideostatsPlaybackUrl videostatsPlaybackUrl;

	@SerializedName("ptrackingUrl")
	private PtrackingUrl ptrackingUrl;

	@SerializedName("atrUrl")
	private AtrUrl atrUrl;

	public void setVideostatsWatchtimeUrl(VideostatsWatchtimeUrl videostatsWatchtimeUrl){
		this.videostatsWatchtimeUrl = videostatsWatchtimeUrl;
	}

	public VideostatsWatchtimeUrl getVideostatsWatchtimeUrl(){
		return videostatsWatchtimeUrl;
	}

	public void setVideostatsDelayplayUrl(VideostatsDelayplayUrl videostatsDelayplayUrl){
		this.videostatsDelayplayUrl = videostatsDelayplayUrl;
	}

	public VideostatsDelayplayUrl getVideostatsDelayplayUrl(){
		return videostatsDelayplayUrl;
	}

	public void setQoeUrl(QoeUrl qoeUrl){
		this.qoeUrl = qoeUrl;
	}

	public QoeUrl getQoeUrl(){
		return qoeUrl;
	}

	public void setYoutubeRemarketingUrl(YoutubeRemarketingUrl youtubeRemarketingUrl){
		this.youtubeRemarketingUrl = youtubeRemarketingUrl;
	}

	public YoutubeRemarketingUrl getYoutubeRemarketingUrl(){
		return youtubeRemarketingUrl;
	}

	public void setGoogleRemarketingUrl(GoogleRemarketingUrl googleRemarketingUrl){
		this.googleRemarketingUrl = googleRemarketingUrl;
	}

	public GoogleRemarketingUrl getGoogleRemarketingUrl(){
		return googleRemarketingUrl;
	}

	public void setSetAwesomeUrl(SetAwesomeUrl setAwesomeUrl){
		this.setAwesomeUrl = setAwesomeUrl;
	}

	public SetAwesomeUrl getSetAwesomeUrl(){
		return setAwesomeUrl;
	}

	public void setVideostatsPlaybackUrl(VideostatsPlaybackUrl videostatsPlaybackUrl){
		this.videostatsPlaybackUrl = videostatsPlaybackUrl;
	}

	public VideostatsPlaybackUrl getVideostatsPlaybackUrl(){
		return videostatsPlaybackUrl;
	}

	public void setPtrackingUrl(PtrackingUrl ptrackingUrl){
		this.ptrackingUrl = ptrackingUrl;
	}

	public PtrackingUrl getPtrackingUrl(){
		return ptrackingUrl;
	}

	public void setAtrUrl(AtrUrl atrUrl){
		this.atrUrl = atrUrl;
	}

	public AtrUrl getAtrUrl(){
		return atrUrl;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/PlaybackTracking.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class PlaybackTracking : Parcelable, Serializable {
    @SerializedName("videostatsWatchtimeUrl")
    var videostatsWatchtimeUrl: VideostatsWatchtimeUrl? = null

    @SerializedName("videostatsDelayplayUrl")
    var videostatsDelayplayUrl: VideostatsDelayplayUrl? = null

    @SerializedName("qoeUrl")
    var qoeUrl: QoeUrl? = null

    @SerializedName("youtubeRemarketingUrl")
    var youtubeRemarketingUrl: YoutubeRemarketingUrl? = null

    @SerializedName("googleRemarketingUrl")
    var googleRemarketingUrl: GoogleRemarketingUrl? = null

    @SerializedName("setAwesomeUrl")
    var setAwesomeUrl: SetAwesomeUrl? = null

    @SerializedName("videostatsPlaybackUrl")
    var videostatsPlaybackUrl: VideostatsPlaybackUrl? = null

    @SerializedName("ptrackingUrl")
    var ptrackingUrl: PtrackingUrl? = null

    @SerializedName("atrUrl")
    var atrUrl: AtrUrl? = null
    override fun toString(): String {
        return "PlaybackTracking{" +
                "videostatsWatchtimeUrl = '" + videostatsWatchtimeUrl + '\'' +
                ",videostatsDelayplayUrl = '" + videostatsDelayplayUrl + '\'' +
                ",qoeUrl = '" + qoeUrl + '\'' +
                ",youtubeRemarketingUrl = '" + youtubeRemarketingUrl + '\'' +
                ",googleRemarketingUrl = '" + googleRemarketingUrl + '\'' +
                ",setAwesomeUrl = '" + setAwesomeUrl + '\'' +
                ",videostatsPlaybackUrl = '" + videostatsPlaybackUrl + '\'' +
                ",ptrackingUrl = '" + ptrackingUrl + '\'' +
                ",atrUrl = '" + atrUrl + '\'' +
                "}"
    }
}